spec/cedar_policy/entities_spec.rb

```
@@ -37,4 +37,36 @@
+            owner: User
+          };
+
+          action View  appliesTo {
+              principal: [User],
+              resource: [Image]
+          };
+        SCHEMA
+      )
+    end
+
+    let(:user) { {uid: {type: "User", id: "1"}, attrs: {}, parents: []} }
```

rubocop (1.68.0)

[convention] Space inside { missing.

(Layout/SpaceInsideHashLiteralBraces)